packages/ckeditor5-engine/src/conversion/upcasthelpers.js

```diff
@@ -330,8 +335,61 @@ export default class UpcastHelpers extends ConversionHelpers {
 	 * @returns {module:engine/conversion/upcasthelpers~UpcastHelpers}
 	 */
 	elementToMarker( config ) {
+		console.warn(
```

Comment:
Missing error documentation. And you should use http://127.0.0.1:8081/ckeditor5/20.0.0/api/module_utils_ckeditorerror.html#static-function-attachLinkToDocumentation